iothub/service/src/ManagedIdentity.cs

```diff
@@ -15,10 +13,21 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class ManagedIdentity
     {
+        /// <summary>
+        /// The user identity resource Id used to access the storage account for import and export jobs.
+        /// </summary>
+        [Obsolete("Use UserAssignedIdentity instead")]
+        [JsonIgnore]
+        public string userAssignedIdentity
```

Comment:
Migrating users to the new, properly-cased property.

Reply:
Ugh....

Reply:
```suggestion
        [EditorBrowsable(EditorBrowsableState.Never)]
        public string userAssignedIdentity
```